Gopkg.toml

```diff
@@ -37,10 +37,6 @@
   name = "github.com/cyverse-de/configurate"
   version = "0.2.3"
 
-[[constraint]]
-  branch = "master"
-  name = "github.com/cyverse-de/vaulter"
```

Comment:
If we're purging vault, does this mean we can archive this repo too?

Reply:
Probably, yeah. We'll want to check for sure nothing else uses it but I can't think of anything.